src/reducers/book.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable arrow-parens */
```

Comment:
I disabled this because it puts a single argument of an arrow function with a direct return between parenthesis and in order for Stickler to pass in the pull request there should be no parenthesis in such case.

Reply:
Okay!
I could see this in one of the previous checks https://github.com/Redvanisation/Bookstore/pull/2/checks?check_run_id=369657956 in this commit https://github.com/Redvanisation/Bookstore/pull/2/commits/292ad577a253cc070451c1e1354def7cd892fc40

I think there should be a rule for it in the config. Thanks for the explanation. :relaxed: